tests/test_args_parser.py

```diff
@@ -199,6 +205,7 @@ def _make_expected_result(
         family,
         gcs_bucket,
         image_name,
+        licenses,
```

Comment:
these tests are failing - not sure what the issue is, but we should make sure these pass

Reply:
we tracked this down to a change between python3.8 and python3.9 - the namespace args are sorted for me on 3.9, so I think this is due to the change in dictionary ordering in python?